### DISCUSSION TITLE
Reading files from .env variables using _FILE convention fails when the file isn't found

### DISCUSSION THREAD

**Discussion by petervukovic** [directus/directus#14624] [Feature Requests]
### Describe the Bug

When .env file variable references a file (using _FILE suffix at the end of the variable name), and such file isn't found, running `directus bootstrap` fails with an unfriendly error.

### To Reproduce

1. Install directus manually per documentation
2. Create a .env file
3. Add the following environment variable `DB_SSL__CA_FILE = "ca_certificate.crt"` to the env file
4. Run `npx directus bootstrap`

**Expected:**
I should get a warning or a clear message explaining the reason for aborting the program execution.

**Actual:**
The program crashes with a hard-to-read error.

### Errors Shown

```
/directus/node_modules/directus/dist/env.js:346
                    throw new Error(`Failed to read value from file "${value}", defined in environment variable "${key}".`);
                    ^

Error: Failed to read value from file "ca_certificate.crt", defined in environment variable "DB_SSL__CA_FILE".
    at processValues (/Users/petervukovic/Repositories/sparwk/directus/node_modules/directus/dist/env.js:346:27)
    at Object.<anonymous> (/Users/petervukovic/Repositories/sparwk/directus/node_modules/directus/dist/env.js:251:7)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1147:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/petervukovic/Repositories/sparwk/directus/node_modules/directus/dist/cache.js:9:31)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)

Node.js v17.0.1
```

### What version of Directus are you using?

9.14.5

### What version of Node.js are you using?

17.0.1

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### How are you deploying Directus?

running locally

**Comment by br41nslug** [comment]
Directus failing when files referenced using the `_FILE` convention are not readable is very much intentional.
> Error: Failed to read value from file "ca_certificate.crt", defined in environment variable "DB_SSL__CA_FILE".

I do see that hiding the stack dump could make it somewhat more readable but i don't personally find the error message above "hard-to-read".
I've converted this to a discussion to keep the conversation open for suggested improvements and community input.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
